lib/game.rb

```diff
@@ -0,0 +1,177 @@
+
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2, :token, :cells, :move, :player
+ 
+
+  
+  WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+  def initialize(player_1 = "X", player_2 = "O", board = [" ", " ", " ", " ", " ", " ", " ", " ", " "])
+    if board == [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+      @board = Board.new
+    else
+      @board = board
+    end
+    if player_1 == "X"
+      @player_1 = Player::Human.new(player_1)
+      @player_1.board = @board
```

Comment:
Seems like a `board` should belong to a `game`, not a `player`. Don't both players use the same board?